cellprofiler/modules/trackobjects.py

```diff
@@ -1032,6 +1108,49 @@ def draw(self, labels, ax, object_numbers):
                 ax.annotate(str(n), xy=(x, y), color='white',
                             arrowprops=dict(visible=False))
 
+    def run_followneighbors(self, workspace, objects):
+        '''Track objects based on following neighbors'''
+        cellstar = NeighbourMovementTracking()
+        cellstar.parameters_tracking["avgCellDiameter"] = self.average_cell_diameter.value
+        cellstar.parameters_tracking["max_distance"] = self.pixel_radius.value
+        multiplier = float(NeighbourMovementTrackingParameters.parameters_cost_iteration["default_empty_cost"]) / \
+                     NeighbourMovementTrackingParameters.parameters_cost_initial["default_empty_cost"]
+        cellstar.parameters_cost_iteration["default_empty_cost"] = multiplier * self.drop_cost.value
+        cellstar.parameters_cost_initial["default_empty_cost"] = self.drop_cost.value
```

Comment:
The idea is that in `NeighbourMovementTracking` there is a separate set of parameters for initial and the following calculations. On trackobjects level we do not want to give that freedom of choice. Therefore we set both parameters using one user specified parameters.

I've refactored it a bit.